helmfile.d/upstream/falcosecurity/falco/values.yaml

```diff
@@ -469,6 +513,14 @@ falcosidekick:
   # -- Listen port. Default value: 2801
   listenPort: ""
 
+# -- Enable the response actions using Falco Talon.
+responseActions:
+  enabled: false
+
+# -- For configuration values, see https://github.com/falcosecurity/charts/blob/master/charts/falco-talon/values.yaml
+# -- It must be used in conjunction with the response_actions.enabled option.
+falco-talon: {}
```

Comment:
This means that falco talon is not installed, right?
For now I think we want to keep it disabled.

Reply:
Ok, thanks. That sounds good.